playbooks/prepare_vm_for_minishift.yml

```diff
@@ -0,0 +1,48 @@
+- hosts: '{{ hosts | default("all") }}'
+  become: '{{ become | default("yes") }}'
+  gather_facts: '{{ gather_facts | default("no") }}'
+  vars:
+    setup_nested_virt: false
+  tasks:
+    - name: "Install python if not installed"
+      raw: test -e /usr/bin/python || sudo yum install -y python
+    - name: "Install docker, net-tools and firewalld"
+      package:
+        name: "{{ item }}"
+        state: present
+      with_items:
+        - docker
+        - net-tools
+        - firewalld
+    - name: "Ensure group docker exists"
```

Comment:
nit: not really a stickler, but maybe space between task could be pleasant to the eye :) but we could do it next time if need be.

Reply:
Yep, it was a bit crowded. Fixed it now.